tpcc/ddl.go

```diff
@@ -126,7 +126,7 @@ CREATE TABLE IF NOT EXISTS warehouse (
 	w_zip CHAR(9),
 	w_tax DECIMAL(4, 4),
 	w_ytd DECIMAL(12, 2),
-	PRIMARY KEY (w_id)
+	PRIMARY KEY (w_id) CLUSTERED
```

Comment:
We may use `/*T![clustered_index] CLUSTERED */` so that these DDLs are compatible with MySQL.

Reply:
Many thanks, has changed to to `/*T![clustered_index] CLUSTERED */`